test/automation/src/positron/positronConsole.ts

```diff
@@ -201,33 +196,46 @@ export class PositronConsole {
 	}
 
 	async waitForInterpreterShutdown() {
-		await this.waitForConsoleContents((contents) =>
-			contents.some((line) => line.includes('shut down successfully'))
-		);
+		await this.waitForConsoleContents('shut down successfully');
 	}
 
 	async doubleClickConsoleText(text: string) {
-		await this.code.driver.page.locator(`${ACTIVE_CONSOLE_INSTANCE} div span`).getByText(text).dblclick();
+		await this.code.driver.page.locator(CONSOLE_LINES).getByText(text).dblclick();
 	}
 
-	async waitForConsoleContents(accept?: (contents: string[]) => boolean) {
-		const elements = await this.code.waitForElements(`${ACTIVE_CONSOLE_INSTANCE} div span`,
-			false,
-			(elements) => accept ? (!!elements && accept(elements.map(e => e.textContent))) : true);
-		return elements.map(e => e.textContent);
+
+	async waitForConsoleContents(
+		consoleText: string,
+		options: {
+			timeout?: number;
+			contain?: boolean;
+			matchCount?: number;
```

Comment:
so contain and matchCount have to be used together, right?  But both are optional?

Reply:
these are the defaults:
```
const { timeout = 15000, contain = true, matchCount = 1 } = options;
```
i just updated this snippet seconds before you commented to make it simpler. let me know if it still is not clear.